tests/Php81/Php81Test.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Polyfill\Tests\Php81;
+
+use PHPUnit\Framework\TestCase;
+
+class Php81Test extends TestCase
+{
+    /**
+     * @covers \Symfony\Polyfill\Php81\Php81::array_is_list
+     */
+    public function testArrayIsList()
+    {
+        $this->assertTrue(array_is_list([]));
+        $this->assertTrue(array_is_list([\NAN]));
```

Comment:
this misses a case of an array with holes, and potentially also of a sequential array not starting from 0 (see the RFC describing this case for rejecting the `is_sequential_array` name).

Maybe we could take test cases from the PHP PR implementing the RFC.

Reply:
updated